examples/webhook.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/xanzy/go-gitlab"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
```

Comment:
Please place external deps in their own block...

```suggestion
import (
	"encoding/json"
	"errors"
	"fmt"
	"io"
	"io/ioutil"
	"net/http"
	
	"github.com/xanzy/go-gitlab"
)
```